Wikipedia/Code/AppearanceSettingsViewController.swift

```diff
@@ -205,6 +205,12 @@ open class AppearanceSettingsViewController: UIViewController, UITableViewDataSo
                 break
             }
             
+        } else if let customViewItem = sections[indexPath.section].items[indexPath.item] as? AppearanceSettingsCustomViewItem {
+            if customViewItem.viewController is ImageDimmingExampleViewController {
+                let bottomView = UIView(frame: CGRect(x: 0, y: cell.frame.height - ImageDimmingExampleViewController.bottomSpacing, width: cell.frame.width, height: ImageDimmingExampleViewController.bottomSpacing))
+                bottomView.backgroundColor = tableView.backgroundColor
+                cell.addSubview(bottomView)
```

Comment:
Usually best to avoid adding subviews in `willDisplay`... could you set the bg color on the white views on the sides of https://phab.wmfusercontent.org/file/data/dblft4xffury3xn2exz4/PHID-FILE-gxcq6s2d73u2s7cpghbc/IMG_0024.PNG ?

Reply:
The bg color is already set. This is adding a view that makes it look like there's spacing between the cells.

Reply:
If you set a breakpoint on line 212 where it's adding this view, then rotate landscape and scroll up and down, it's hitting that breakpoint each time you scroll.

Reply:
It's also off when initially rotating from portrait to landscape:

<img width="723" alt="screen shot 2017-09-28 at 4 23 17 pm" src="https://user-images.githubusercontent.com/3143487/30994588-9af3b754-a469-11e7-973b-a5df6435b603.png">

Reply:
> The bg color is already set. This is adding a view that makes it look like there's spacing between the cells.

Here's the view I'm talking about:

![screen shot 2017-09-28 at 4 43 40 pm](https://user-images.githubusercontent.com/3143487/30995061-99b37b6a-a46c-11e7-840e-66a1bfb46e47.png)

Looks like the spacing is already there, the view is just the wrong color:

![screen shot 2017-09-28 at 4 40 25 pm](https://user-images.githubusercontent.com/3143487/30995060-999450f0-a46c-11e7-959d-b5d518ee844a.png)

Reply:
Oh I see now - the lighter color extends to the edges.

I'd maybe try adding a view to ImageDimmingExampleViewController.xib for the lighter color, then set that view's color in ImageDimmingExampleViewController.swift's `apply(theme: Theme)`.